### ISSUE TITLE
How you used MobileNet model? 

### ISSUE THREAD

**Issue by meet-minimalist** [Adamdad/keras-YOLOv3-mobilenet#40]
Have you used the pretrained MobileNet model and construct yolo layers on top? And used pretrained MobileNet weights for Mobilenet initialization? 

Or you just took the architecture of MobileNet and add yolo layers and started training from scratch using voc or coco?

**Comment by ZH-Lee** [comment]
I used MobileNet weights for MobileNet initialization, and feed VOC2007 datasets into network. But, i got train loss non-convergence（train loss : 30, val loss:26），whaterver i decrease learning rate, i can't got train loss convergence.

**Comment by meet-minimalist** [comment]
@ZH-Lee so what you did to make it converge?

**Comment by ZH-Lee** [comment]
> @ZH-Lee so what you did to make it converge?

i didn't make it converge. maybe the datasets is small??? I haven't solve the problem.

**Comment by meet-minimalist** [comment]
@ZH-Lee ohk. 👍

**Comment by ruoruo6** [comment]
> 
> 
> I used MobileNet weights for MobileNet initialization, and feed VOC2007 datasets into network. But, i got train loss non-convergence（train loss : 30, val loss:26），whaterver i decrease learning rate, i can't got train loss convergence.

How do you train with VOC dataset? just transfer voc format to coco in this code? i attemp to do it but the map is much low than yolov2 on voc. Why ?
